components/resources/wallet_strings.grdp

```diff
@@ -1072,4 +1072,7 @@
   <message name="IDS_BRAVE_WALLET_PROCEED_BUTTON" desc="Proceed action button text">Proceed</message>
   <message name="IDS_BRAVE_WALLET_FOUND_ISSUES" desc="Notice that a number of issues were found">We found <ph name="VALUE">$1<ex>2</ex></ph> issues.</message>
   <message name="IDS_BRAVE_WALLET_NO_ROUTES_FOUND" desc="Error message for when no swap routes could be found">No routes found</message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_NAME_TOO_LONG_ERROR" desc="An error that appears when the user enters an account name that is too long">Account name must be <ph name="VALUE">$1<ex>30</ex></ph> characters or less</message>
```

Comment:
should we remove this `30` here?

Reply:
@thypon what do you suggest we change it to? The 30 is wrapped in the <ex> tag to convey that it is an example value

Reply:
Is `$1` gonna be replaced with `30` and this will be `<ph>30<ex>30</ex></ph`?
I might have misunderstood this.

In case it's not, the second 30 should be `$2` and passed as a variable

Reply:
yes `$1` is meant to be replaced by the current max (`30`)

Reply:
I'd leave only the current constant here, without mentioning the other one. It does not look to add anything extra here.

[this is not a security consideration]

Reply:
`<ex>30</ex>` is meant to provide a hint as to what the replacement text should look like. 
cc: @mkarolin for your thoughts